_episodes/14.md

```diff
@@ -0,0 +1,330 @@
+March of the Trinos! Be careful Commander Bun Bun! That Iceberg doesn't look stable!<br/>
+<a href="https://joshdata.me/iceberger.html">https://joshdata.me/iceberger.html</a>
+</p>
+
+## Iceberg links
+
+ * [Apache Iceberg](https://iceberg.apache.org/)
+ * [Apache Iceberg Community](https://iceberg.apache.org/community/)
+
+## Guests
+  
+ * David Phillips, creator of Trino and Presto, and CTO at 
+ [Starburst](https://starburst.io) ([@electrum32](https://twitter.com/electrum32))
+
+## Release 354
+
+Release notes discussed: <https://trino.io/docs/current/release/release-354.html>
+
+Martin’s list:
+ - Support for OAuth 2.0 in CLI
+ - Support for MemSQL 3.2
+ - Pushdown of ORDER BY ... LIMIT for MemSQL, MySQL and SQL Server connectors
+ - Support for time(p) in SQL Server
+
+Manfred’s notes:
+ - LEFT, RIGHT and FULL JOIN 
+ - Preferred write partitioning on by default (needs statistics)
+ - Small but useful fix on Elasticsearch (single value array)
+ - Hive connector
+ - Fix ACID table DELETE and UPDATE - critical fix is in! Boom!
+ - Avro format improvement
+ - CSV and Glue metadata improvement
+ - Iceberg - date and timestamp improvement
+ - CREATE SCHEMA fixes  in MySQL, PostgreSQL, Redshift and SQL Server
+ - Bunch of other fixes in those connectors
+
+## Concept of the week: Apache Iceberg and the table format
+
+### The Hive table format
+
+For the last decade or so, big data professionals' only option to query their 
+data was to, in some way shape or form, use the Hive model. The Hive model is
+very simple, but it enabled running queries over files in a distributed file
+system. 
+
+To accomplish this, Hive uses a metastore service which stores and manages
+metadata. For Hive and Trino, this metadata acts as a pointer to the files
+containing the data, contains the file format, and has the column structure and
+types. This enabled Hive to query the correct files and data within those files
+for a SQL query. For more information on Hive's architecture, read the
+[Gentle Introduction to Hive](/blog/2020/10/20/intro-to-hive-connector.html)
+blog. After the initial model gained adoption, Hive added other features such as
+partitioning, which used the directory structures of the filesystems to split
+the files of data partitioned on a special column into different directories.
+We talk about this in more depth [a few episodes back](/episodes/5.html).
+
+This solved some initial issues facing engineers in big data, but there were
+quite a few issues with this model. The Hive model is very rigid and not able
+to adapt to your needs as requirements change. For example, if you started
+partitioning your data splitting by date and segmenting by month, that table is
+stuck with that partitioning forever. The only way to update it is to create a
+new table with your new partition values, and migrate all of your data from
+the old table to the new table. Another issue stems from the separation of data
```

Comment:
to the new table. With the common data sizes such a migration is often a long process, sometimes even impossible. Another issue...